docs/development/style-guide.rst

```diff
@@ -194,8 +215,8 @@ Time and Date
 =============
 
 Use numerals when exact times are expressed, followed by *ante meridiem* or
-*post meridiem* abbreviated in lowercase with periods, or use the twenty-four-
-hour system.
+*post meridiem* abbreviated in lowercase with periods, or use the 24-hour
```

Comment:
Can we not just use 24h style? It is the standard everywhere except North America. Certainly, no astronomy journal would accept AM/PM.

Reply:
:100: for 24hr only without the added confusion of AM/PM. I syill have issues with 12:30AM is night or just after noon, and I'm living surrounded with weird units.

Reply:
You could mention 24hr time first and say that we prefer the 24hr format in general, but I think it's fine to clarify how to format AM/PM as well.

Reply:
@pllim - No, `Time` cannot do AM/PM!

I do think a style guide should give a single preference, not multiple (obligatory zen of python reference: "There should be one-- and preferably only one --obvious way to do it.")

Reply:
So it sounds like since ``Time`` cannot do AM/PM, the consensus is to remove the guideline regarding AM/PM and recommend using 24hr style? Moving forward with copy editing I can also edit any instances of AM/PM I find to reflect 24hr instead.